build/config/ios/rules.gni

```diff
@@ -131,7 +135,7 @@ template("ios_app") {
   copy_gen_target_name = target_name + "_copy"
   copy(copy_gen_target_name) {
     sources = [
-      "$root_build_dir/Info.plist",
+      "$root_build_dir/plist/$app_name/Info.plist",
```

Comment:
Does it matter if we are still giving the intermediate plist files from each target their own directory?